packages/cozy-intent/src/api/models/methods.ts

```diff
@@ -1,4 +1,6 @@
+import { AppManifest } from './applications'
+
 export type NativeMethodsRegister = {
   logout: () => Promise<void>
-  openApp: (href: string) => Promise<void>
+  openApp: (href: string, app: AppManifest) => Promise<void>
```

Comment:
I prefer to have an object as argument instead of having two args : `openApp: ({href, app})`. Because I don't like to write "openApp(null, {...})". And pretty sure we'll start writing this kind of code pretty soon

Reply:
I was not confident enough to remove the href param, but I have the intuition it will be obsolete soon.

I should have a better understanding of possible use cases after I implement the cozy-store connectors link. I propose to do the refactoring after that.